src/main/java/chatbot/Alfred.java

```diff
@@ -0,0 +1,315 @@
+package chatbot;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class Alfred {
+    protected static final String LINE = "________________________________________________________________________________________";
+    protected static ArrayList<Task> list = new ArrayList<Task>();
+
+    public Alfred(String data) {
+        java.io.File file;
+        try {
+            file = new java.io.File(data);
+            this.processData(file);
+        } catch (FileNotFoundException e) {
+            this.echo("Sorry Master Bruce. I cannot find the data. I will create a new file for you.");
+            // create a new file in the same location
+            file = new java.io.File("data/alfred.txt");
+        }
+    }
+    public void processData(File file) throws FileNotFoundException {
```

Comment:
Very long method, maybe you can abstract out some parts into functions or add more comments?